src/tests/ftest/util/AgentUtils.py

```diff
@@ -111,11 +164,29 @@ def stop_agent(client_list):
 
     client_list -- kill daos_agent on these hosts
     """
-
+    global sessions
```

Comment:
(pylint-global-variable-not-assigned) Using global for 'sessions' but no assignment is done

(pylint-invalid-name) Invalid constant name "sessions"